### PR TITLE
Created basic driver for H-bridge for magnetorquer

### LINKED ISSUE #295: AVIONICS: Design and Write H-Bridge Driver

### PR DESCRIPTION
**Pull request by Shade-MC**
Fixes #295

### CODE REVIEW

**Comment by WattsUp** [comment]
The class you wrote isn't the magnetorquer class more the H bridge driver. I recommend you have a class just for the H bridge (called DRV8837) and have a separate class that encapsulates the h bridge plus the ADC that will measure the coil's current (Magnetorquer). The PMIC does this with PowerNode under components folder that uses several drivers https://github.com/CougsInSpace/CougSat1-Software/tree/master/CougSat1-PMIC/src.

Follow the ADC class as an example for formatting: https://github.com/CougsInSpace/CougSat1-Software/blob/master/CougSat1-PMIC/src/drivers/LTC2499.cpp
Your constuctor creates the PWM and digitalOut objects but they get freed once it leaves scope. Those three objects should be private member variables.
The setCoil function does not work as you hope, when you set the PWM of the reverse pin, you don't reset the forward pin to zero. See section 7.3.1 in the datasheet DRV883x
The wake function should set the PWMs to 0 
The default period for PWM out is 20ms (50Hz) I expect you want this  higher (you can calculate it if you want from the coil's inductance).

**Comment by Shade-MC** [comment]
I think i fixed the issue that were found

File: CougSat1-ADCS/src/drivers/DRV8837.h [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+#include <mbed.h>
+#include "ADCSPins.h"
+
+class DRV8837
+{
+  public:
+    DRV8837(PinName forPin, PinName revPin, PinName pSleep);
+    unit8_t setCoil(float input);
+    unit8_t setCoil(float input, float time);
```
Comment (WattsUp) [inline CougSat1-ADCS/src/drivers/DRV8837.h]: Is this not implemented?

Comment (Shade-MC) [reply CougSat1-ADCS/src/drivers/DRV8837.h]: This just adds the DRV8837 and LTC2499 drivers to the ADCS branch.
I also deleted the unused setCoil function, we decided to implement it's functionality in the event queue system

Comment (WattsUp) [reply CougSat1-ADCS/src/drivers/DRV8837.h]: Rename setCoil to setOutput and I'll approve

**Pull request merged**